plugins/migration/bin/fix-imports.js

```diff
@@ -33,6 +33,7 @@ export const fixImports = createSubCommand(
       const newContents = createNewContents(contents);
       const inPlace = this.opts().inPlace;
 
+
```

Comment:
[The styleguide says](https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace) that multiple blank lines are "not encouraged".

Reply:
Whoops! thanks for the catch.